starknet-rpc-test/get_transaction_receipt.rs

```diff
@@ -156,18 +152,18 @@ async fn work_with_declare_transaction(madara: &ThreadSafeMadaraClient) -> Resul
 
     let fee_token_address = FieldElement::from_hex_be(FEE_TOKEN_ADDRESS).unwrap();
     let expected_fee =
-        FieldElement::from_hex_be("0x000000000000000000000000000000000000000000000000000000000000d3ae").unwrap();
+        FieldElement::from_hex_be("0x00000000000000000000000000000000000000000000000000000000000030fc").unwrap();
 
     assert_declare_tx_receipt(
         declare_tx_receipt,
         DeclareTransactionReceipt {
             transaction_hash: FieldElement::from_hex_be(
-                "0x05e0f64e8140019f2657f244dd9fd136d18acc6f52d8a0b85d3f84a110d4c708",
+                "0x012fe0d3db3b6e1b80839c187adf3b03d8868048e77acb6d39bcdae4af087fe2",
             )
             .unwrap(),
             actual_fee: expected_fee,
             finality_status: TransactionFinalityStatus::AcceptedOnL2,
-            block_hash: FieldElement::from_hex_be("0x031622c96d67dabe52c0317752d6e6be69a4288e6dcec09a6f8324bee49d4ce5")
+            block_hash: FieldElement::from_hex_be("0x001378ccd5b2e291783b670db2fca8477abbc3ba3cae78ca867282102eb06b1f")
```

Comment:
Don't assert block hash. It's not reproductible because the tests can happen in any order